rollup.globals.json

```diff
@@ -36,5 +36,6 @@
 	"@tradeshift/elements.select": "ts.elements.select",
 	"@tradeshift/elements.tabs": "ts.elements.tabs",
 	"@tradeshift/elements.tab": "ts.elements.tab",
-	"@tradeshift/elements.text-field": "ts.elements.text-field"
+	"@tradeshift/elements.text-field": "ts.elements.text-field",
+	"@tradeshift/elements.confirmation-prompt": "ts.elements.confirmation-prompt"
```

Comment:
Where does the confirmation-prompt come from? I don't see the code for it in this PR.

Reply:
Somehow I missed a commit :)))